hooks/configure

```diff
@@ -74,6 +74,24 @@ switch_service() {
 	esac
 }
 
+update_pi_config_value() {
```

Comment:
I think this could be written in Python. The sed parts below just make me scratch my head. It's probably 100% correct and it would just be some extra work but I would feel far more confident this way.